### DISCUSSION TITLE
how to add motif sequence in plotFootprints

### DISCUSSION THREAD

**Discussion by lilwo** [GreenleafLab/ArchR#975] [Questions / Documentation]
In the paper "ArchR is a scalable software package for integrative single-cell chromatin accessibility analysis"
fig4i–k, have motif sequence in the topleft
I found the code "ArchR_PostAnalysis_Large_Heme_Analysis.R
seFoot_None <- plotFootprints(
  ArchRProj = proj, 
  positions = motifPositions[stringr::str_split(colnames(p),pattern=" ", simplify=TRUE)[,1]], 
  groupBy = "ClustFinal",
  normMethod = "none",
  plotName = "Footprints-No-Normalization"
)

seFoot_None2 <- plotFootprints(
  ArchRProj = proj, 
  inputSE = seFoot_None,
  positions = motifPositions[stringr::str_split(colnames(p),pattern=" ", simplify=TRUE)[,1]], 
  groupBy = "ClustFinal",
  normMethod = "none",
  plotName = "Footprints-No-Normalization",
  smoothWindow = NULL
)

but the new  ArchR ,plotFootprints doesn't have the inputSE, and I can't  find any parameter about adding motif sequence
so is there any way to add motif sequence?

**Comment by rcorces** [comment]
Those motif PWM images were added manually using Adobe Illustrator. We do not provide code to generate images of motif PWMs.

**Comment by lilwo** [comment]
Thanks a lot



---Original---
From: "Ryan ***@***.***&gt;
Date: Fri, Aug 13, 2021 23:18 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [GreenleafLab/ArchR] how to add motif sequence in plotFootprints (#975)




 
Those motif PWM images were added manually using Adobe Illustrator. We do not provide code to generate images of motif PWMs.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
